openstack/workflow/v2/crontriggers/doc.go

```diff
@@ -4,38 +4,70 @@ Package crontriggers provides interaction with the cron triggers API in the Open
 Cron trigger is an object that allows to run Mistral workflows according to a time pattern (Unix crontab patterns format).
 Once a trigger is created it will run a specified workflow according to its properties: pattern, first_execution_time and remaining_executions.
 
-Example to list cron triggers
+List cron triggers
+
+To filter cron triggers from a list request, you can use advanced filters with special FilterType to check for equality, non equality, values greater or lower, etc.
+Default Filter checks equality, but you can override it with provided filter type.
 
 	listOpts := crontriggers.ListOpts{
-		WorkflowID: "w1",
+		WorkflowName: &executions.ListFilter{
+			Value: "Workflow1,Workflow2",
+			Filter: executions.FilterIN,
+		},
+		CreatedAt: &executions.ListDateFilter{
+			Value: time.Date(2018, time.January, 1, 0, 0, 0, 0, time.UTC),
+			Filter: executions.FilterGTE,
+		},
 	}
+
 	allPages, err := crontriggers.List(mistralClient, listOpts).AllPages()
 	if err != nil {
 		panic(err)
 	}
+
 	allCrontriggers, err := crontriggers.ExtractCronTriggers(allPages)
 	if err != nil {
 		panic(err)
 	}
-	for _, ex := range allCrontriggers {
-		fmt.Printf("%+v\n", ex)
+
+	for _, ct := range allCrontriggers {
+		fmt.Printf("%+v\n", ct)
 	}
 
-Example to create a cron trigger
+Create a cron trigger. This example will start the workflow "echo" each day at 8am, and it will end after 10 executions.
 
 	createOpts := &crontriggers.CreateOpts{
-		WorkflowID:     "w1",
+		Name:					"daily",
+		Pattern:				"0 8 * * *",
+		WorkflowName:			"echo",
+		RemainingExecutions:	10,
```

Comment:
Nit: the formatting of the above lines is still a little off. This is non-blocking, though.